### PR TITLE
Add page to see & delete plugins that failed to load; refactor ToolbarButtons

### PR DESCRIPTION
**Pull request by Vendicated**

### CODE REVIEW

File: Aliucord/src/main/java/com/aliucord/widgets/FailedPluginWidget.kt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+
+            setBackgroundColor(ColorCompat.getThemedColor(ctx, R.b.colorBackgroundSecondaryAlt))
+            mFilenameView = TextView(ctx, null, 0, R.i.UiKit_TextView_Bold).apply {
+                gravity = Gravity.CENTER_VERTICAL
+                layoutParams = LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.MATCH_PARENT).apply {
+                    weight = 1f
+                }
+
+                text = file.name + " (${FileUtilsKt.getSizeSubtitle(file.length())})"
```
Comment (zt64) [inline Aliucord/src/main/java/com/aliucord/widgets/FailedPluginWidget.kt:64]: You will use string template

**Review by ItzOnlyAnimal — COMMENTED** [review]
plugin errors still sounds weird but i have no better name lol good

**Review by zt64 — CHANGES_REQUESTED** [review]
Instead of hard coding IDs, maybe they could all be moved into one object that contains the IDs, would make porting versions quicker and allow plugins to reuse the ID

**Comment by zt64** [comment]
:trollface:

**Comment by ItzOnlyAnimal** [comment]
:trollface:

**Pull request merged**